keras/wrappers/scikit_learn.py

```diff
@@ -6,6 +6,18 @@
 from ..utils.np_utils import to_categorical
 
 
+class Hideout(object):
```

Comment:
The name is not important. The fact that the class attribute "model" has a get_params method is the problem here. 

Notice that "compiled_model_" doesn't have that problem because that attribute is not returned by the BaseWrapper get_params method, and therefore scikit learn will not look at it.